web/packages/plugins/form/src/fields/ArrayField.tsx

```diff
@@ -42,6 +47,7 @@ export default function Fields(props: any) {
   const { namePath, displayLabel, type, uiAttribute } = props
 
   const { documentId, dataSourceId, onOpen } = useRegistryContext()
+  // @ts-ignore
```

Comment:
the type error should be fixed in master branch, and not necessary to add the @ts-ignore